CHANGELOG.md

```diff
@@ -69,6 +69,7 @@
 
 ### v1.1.0 (TBA)
 #### General
+* Fix #144: Den Namen "Gomez'Rüstung" korrigiert zu "Gomez' Rüstung".
```

Comment:
I noticed that we have started using full sentences and sentences without a clear subject interchangeably throughout the changelog. For example, this line could also be "Der Name "Gomez'Rüstung" ist nun korrigiert zu "Gomez' Rüstung". I don't think it's a big deal, I just wanted to mention it.

Reply:
Otherwise, this PR looks great!

Reply:
> I noticed that we have started using full sentences and sentences without a clear subject interchangeably throughout the changelog.

Yes, I also noticed that you changed my changelog entries. 😄 
For typos the short sentence is absolutely sufficient in my opinion. Also I don't want to change every changelog entry in the comments.

Reply:
Actually you're right. After giving it more thought I will update my changelog entries.